AndroidRunner/Plugins/trepn/README_Trepn.md

```diff
@@ -0,0 +1,13 @@
+# Trepn Plugin
+This plugin collects data via the Trepn profiler, e.g., power consumption, battery temperature, CPUs frequency.
+
+## Configuration
+Below an example configuration is found:
+```json
+  "profilers": {
+    "trepn": {
+      "sample_interval": 100,
+      "data_points": ["battery_power", "mem_usage"]
```

Comment:
Somewhere in the code of the plugin you will find the full list of data points supported by Trepn.

Reply:
Since it's quite a big list, I linked to the json itself